docs/i18n/overview.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: i18n Overview
+name: Overview
+parent: i18n
+nav_order: 2
+---
+
+# Overview i18n Overview
+Many companies use SAP systems and applications and many different countries.
+To ensure easy translation, the "i18n" (Internationalization) process is used.
+The app is not always recreated, but only the texts are translated and replaced.
+
+## Language Tags
+To identify the individual languages, so-called "Language Tags" are used which are defined by [BCP-47] (http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry).
+These language tags are stored in the SAP system in table T002 for all stored languages.
+These correspond to the ["ISO 639 alpha-2 language code"](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes). The SAPUI5 application then converts the determined language key into a BCP-47 tag.
+
+> As a best practice, it should be defined that the BCP-47 language code
+> should be kept as short as possible. When creating it, you should
+> avoid regions, script or subtags as far as possible - unless you add
+> important and helpful information.
+> Good example:
```

Comment:
this looks as if a list is wanted, so "-" it :)
might also make them md formatted code words via backslashes, à la `i18n.properties`

Reply:
ok, i tried to improve it